test/remote/run-e2e-remote.sh

```diff
@@ -35,7 +35,7 @@ export ARGOCD_E2E_SKIP_OPENSHIFT=${ARGOCD_E2E_SKIP_OPENSHIFT:-false}
 # Skip Helm tests
 export ARGOCD_E2E_SKIP_HELM=${ARGOCD_E2E_SKIP_HELM:-false}
 # Skip Helm v2 related tests
-export ARGOCD_E2E_SKIP_HELM2=${ARGOCD_E2E_SKIP_HELM2:-false}
+export ARGOCD_E2E_SKIP_HELM2=${ARGOCD_E2E_SKIP_HELM2:-true}
```

Comment:
Can't this be removed now that those e2e tests have been removed?

Reply:
:+1: